daktari/check.py

```diff
@@ -39,7 +39,21 @@ def verify(self, passed: bool, dualMessage: str) -> CheckResult:
         else:
             return self.failed(pattern.sub(" not ", dualMessage))
 
-    def validate_version(
+    def validate_required_version(
+        self, application: str, installed_version: Union[float, str, None], required_version: Union[float, str, None]
```

Comment:
I guess `required_version` can be `Union[float, str]` since we're not passing `None` ever?